### ISSUE TITLE
Hoping to see this get a port to 2.0? Kind regards sgamez.

### ISSUE THREAD

**Issue by SGZ-Creations** [Yousei9/Logistic-Train-Network#347]
**TL;DR**
After playing a bit on 2.0 space age i found that LTN is really missed on my end as of the simplicity to request recourses to a singular station without having to know much logic at all. I know 2.0 have improve the trains in general but still don't see it as good as LTN is.

**What ?**
![image](https://github.com/user-attachments/assets/c36e7c40-1ea9-4108-bef9-75a505c2eb81)
Simply all ITEM stations that I have build would simply be a singular station that i can request to. helping me craft the things i want with resonnable space needed. Rather than overbuilding which is annoying.

**Why ?**
Simplifies train by 100% for me and i don't see a future without it.

**Comment by SAM105GA** [comment]
点赞

**Comment by SAM105GA** [comment]
give the thumbs-up. Hope to update, LTN is the best MOD...

**Comment by MatinDeWet** [comment]
I would love to see this updated for factorio 2.0

**Comment by AllenJB** [comment]
See previous issues: https://github.com/0ptera/Logistic-Train-Network/issues/346 and https://github.com/0ptera/Logistic-Train-Network/issues/345 (specifically: https://github.com/0ptera/Logistic-Train-Network/issues/345#issuecomment-2428770592 )

TLDR: There's no plans to update LTN to 2.0 - the author no longer plays and the license prevents forking. Similar things can be achieved with the Project Cybersyn mod (which has already been updated to 2.0) and/or the new interrupts and new (parameterized) signals (search the official game forums and subreddit).
